src/main/java/appeng/client/render/effects/LightningFX.java

```diff
@@ -33,12 +33,12 @@
 {
 
 	private static final Random RANDOM_GENERATOR = new Random();
-	final int steps = this.getSteps();
-	final double[][] Steps;
+	private final int steps = 5;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1170)